src/backend/tests/Fusion.Resources.Api.Tests/IntegrationTests/InternalResourceAllocationRequestTests.cs

```diff
@@ -947,6 +948,64 @@ public async Task UpdateRequest_ProposedPersonShouldNotBeNullable_WhenApproving(
             );
             response.Should().BeBadRequest();
         }
+
```

Comment:
whenever possible. add both negative and positive tests

Reply:
Positive cases should already be covered by existing tests.